packages/ws/tsup.config.ts

```diff
@@ -0,0 +1,3 @@
+import { createTsupConfig } from '../../tsup.config';
+
+export default createTsupConfig({ entry: ['./src/**/*.ts'], bundle: false });
```

Comment:
This needs updating to account for #8324

Reply:
I'm aware. Waiting on some stuff from Crawl and I'll do it afterwards